src/backend/api/chat/api.ts

```diff
@@ -0,0 +1,66 @@
+import { appInstances } from "@wix/app-management";
+import { getBehaviorDirective } from "../../database";
+import env from "../../env";
+import OpenAI from "openai";
+
+export interface ChatMessage {
+  author: "Business Buddy" | "User";
+  text: string;
+}
+
+export interface ChatRequestBody {
+  messages: ChatMessage[];
+  product: string;
+}
+
+export async function POST(req: Request) {
+  try {
+    const { instance } = await appInstances.getAppInstance();
+    const { messages, product } = (await req.json()) as ChatRequestBody;
+
+    if (!instance?.instanceId) {
+      return new Response("Unauthorized", { status: 401 });
+    }
+
+    const openai = new OpenAI({
+      apiKey: env.OPENAI_API_KEY,
+    });
+
+    const behaviorDirective = await getBehaviorDirective(instance?.instanceId);
```

Comment:
Can we remove the optional chaining operator? the condition above should help TS understand `instance` exist here.
```suggestion
    const behaviorDirective = await getBehaviorDirective(instance.instanceId);
```